libhb/common.c

```diff
@@ -1790,12 +1790,14 @@ int hb_mixdown_has_codec_support(int mixdown, uint32_t codec)
         case HB_ACODEC_FFFLAC:
         case HB_ACODEC_FFFLAC24:
         case HB_ACODEC_OPUS:
+        case HB_ACODEC_CA_AAC:
+        case HB_ACODEC_FDK_AAC:
```

Comment:
FDK AAC doesn't support 6.1.  So it needs to be separated out as it's own case and return FALSE for 6.1 and TRUE for everything else. Also FDK HE-AAC (HB_ACODEC_FDK_HAAC) supports the same as FDK AAC.